pubspec.yaml

```diff
@@ -0,0 +1,11 @@
+name: lints
+description: Lints that represent the Dart style guide
+version: 0.1.0
+homepage: https://dart.dev/guides/language/effective-dart
```

Comment:
Will we be adding documentation of these rulesets to effective dart?  If not, maybe a new targeted page?

> https://dart.dev/guides/language/lints (or linting)

?

Reply:
My preference would be that effective Dart does somehow communicate what is scoring and what is recommended lints. Is that an option @munificent ?

Reply:
SGTM.  Bob?

Reply:
Mentioning these on "Effective Dart" SGTM.